config/app.default.php

```diff
@@ -10,7 +10,7 @@
      * true: Errors and warnings shown.
      */
     'debug' => filter_var(env('DEBUG', true), FILTER_VALIDATE_BOOLEAN),
-    'isDevelopment' => false,
+    'isDevelopment' =>filter_var(env('DEBUG', true), FILTER_VALIDATE_BOOLEAN),
```

Comment:
I personally recommend always having it default to 'production'/'non-dev' instead. It is less likely to cause accident resource/data leaking on a prod site.

Reply:
Oh, good catch. Fixed!